### PR TITLE
[18.0][MIG] web_pivot_computed_measure: Migration to 18.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT51903

ping @chienandalu @carlos-lopez-tecnativa

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-test-helper
```
Comment (chienandalu) [inline test-requirements.txt:1]: Maybe this should go in a commit appart from the migration one

**Review by chienandalu — APPROVED** [review]
Good work! Tested and looking good. Just a tiny doubt

File: test-requirements.txt (hunk 1 above)
Comment (CarlosRoca13) [reply test-requirements.txt:1]: @pedrobaeza Can you show us the way?

Comment (pedrobaeza) [reply test-requirements.txt:1]: There's no problem in going on the same one, as in future migrations, the filter directory git command strips it out, and in fact it's somehow part of the migration. The pity is that this is not automatizable.

**Review by carlos-lopez-tecnativa — CHANGES_REQUESTED** [review]
I tested the module, but I believe something isn’t working correctly. When grouping by a Many2one field (e.g., by Customer or Salesperson) in the columns:

Without a computed measure, the values are displayed correctly:
image

However, with a computed measure (e.g., Total - Taxes), the values appear empty:
image

Could you please check this?

**Comment by CarlosRoca13** [comment]
@carlos-lopez-tecnativa thanks 4 the review :) It should be solved right now

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM

**Pull request merged**